### ISSUE TITLE
Create an example app

### ISSUE THREAD

**Issue by andypike** [andypike/rectify#2]
Create an app that demonstrates how to use Rectify. It shouldn't be a basic CRUD app as that will just make Rectify look like overkill (which it is for a CRUD app). 

At the same time it can't be crazy complicated otherwise developers may find it hard to follow.

**Comment by andypike** [comment]
I've made a rough start in this here: https://github.com/andypike/smarty_pants

**Comment by jurgens** [comment]
@andypike thanks, this example app has answered couple of my questions regarding the use of rectify
